src/scenes/FunderHome/index.tsx

```diff
@@ -25,10 +52,16 @@ export default function FunderHome() {
       {
         id: 'report',
         label: strings.REPORT,
-        children: <FunderReportView />,
+        children: (
+          <FunderReportView
+            selectedProjectId={selectedProjectId}
+            reports={reports}
+            userFundingEntity={userFundingEntity}
+          />
+        ),
       },
     ];
-  }, [activeLocale]);
+  }, [activeLocale, userFundingEntity]);
```

Comment:
Should we add the selectedProjectId here?

Reply:
Good catch!